client/src/index.css

```diff
@@ -26,10 +26,6 @@ button, input[type=text], input[type=button] {
 }
 
 :root {
-  -moz-user-select: none;
-  -webkit-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
```

Comment:
This should not be removed. Please apply `user-select: text;` on elements that you wish to be selectable, but not removing the global settings.